internal/service/advancedcluster/data_source_advanced_cluster.go

```diff
@@ -208,7 +208,7 @@ func DataSource() *schema.Resource {
 						},
 					},
 				},
-				Set: replicationSpecsHashSet,
+				// Set: replicationSpecsHashSet,
```

Comment:
let's just remove this (I know in the resource it is commented, but really don't know why 😃)